doc/source/whatsnew/v0.7.3.rst

```diff
@@ -28,7 +21,7 @@ New features
 .. code-block:: python
 
    from pandas.tools.plotting import scatter_matrix
-   scatter_matrix(df, alpha=0.2)
+   scatter_matrix(df, alpha=0.2)        # noqa F821
```

Comment:
Why is the noqa needed here?

Reply:
I think the other alternative is to create `df`, but it may be tricky to make the data match the plot saved in the image, and this is the release notes of pandas 0.7, so wouldn't spend much time on that.